<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userInfo1</class>
 <widget class="QMainWindow" name="userInfo1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	background-image: url(:/xiuxian/images/backgroundlogin.jpg);
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/xiuxian/images/shopping-bag.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="userBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>370</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/xiuxian/images/user.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chatBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/xiuxian/images/message-circle.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/xiuxian/images/min.png);
	border:none;
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	image: url(:/xiuxian/images/close.png);
	border:none;
	background:transparent;
}
QPushButton:pressed{
	background-color:#405361;
	border-style:inset;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>19</y>
       <width>601</width>
       <height>331</height>
      </rect>
     </property>
     <widget class="QFrame" name="chatFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>601</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	background:transparent;
	border:none;
}
QLineEdit{
	border:none;
	background-color:rgb(85, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="chatList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>431</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	font-size:18px;
	color: rgb(255, 255, 127);
}
QScrollBar:vertical
{
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px;
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%); 
    border-radius:4px;
    min-height:20;
}

QScrollBar::sub-line:vertical
{
    height:9px;width:8px;
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
    height:9px;width:8px;
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
    height:9px;width:8px;
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical 
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}

</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="msgEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>260</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="backFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="backList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>591</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	font-size:18px;
	color: rgb(255, 255, 127);
}
QScrollBar:vertical
{
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px;
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%); 
    border-radius:4px;
    min-height:20;
}

QScrollBar::sub-line:vertical
{
    height:9px;width:8px;
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
    height:9px;width:8px;
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
    height:9px;width:8px;
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical 
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>*标记为已经装备</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="userFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>601</width>
        <height>321</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="userinfolabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
color:rgb(255, 255, 127);
font-size:30px;
</string>
       </property>
       <property name="text">
        <string>userInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="accountLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
color:rgb(255, 255, 127);
font-size:30px;
</string>
       </property>
       <property name="text">
        <string>userInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="acc">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
color:rgb(255, 255, 127);
font-size:30px;
</string>
       </property>
       <property name="text">
        <string>account:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="acc_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
color:rgb(255, 255, 127);
font-size:30px;
</string>
       </property>
       <property name="text">
        <string>username：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nameLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
color:rgb(255, 255, 127);
font-size:30px;
</string>
       </property>
       <property name="text">
        <string>username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="setNameBtn">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>160</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:14px</string>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameEdit">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>220</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="nameBtn">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>220</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
     <zorder>backFrame</zorder>
     <zorder>userFrame</zorder>
     <zorder>chatFrame</zorder>
    </widget>
    <widget class="QLabel" name="IsConnected">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
